framework/views/errorHandler/callStackItem.php

```diff
@@ -10,35 +10,49 @@
 /* @var $args array */
 /* @var $handler \yii\web\ErrorHandler */
 ?>
-<li class="<?php if ($index === 1 || !$handler->isCoreFile($file)) echo 'application'; ?> call-stack-item"
```

Comment:
This is what `PHPCBF` thinks it should look like. Haven't reviewed this myself yet.